src/tests/ftest/util/daos_utils.py

```diff
@@ -377,39 +375,83 @@ def container_create_snap(self, pool, cont, snap_name=None, epoch=None,
                 Defaults to None.
 
         Returns:
-            CmdResult: Object that contains exit status, stdout, and other
-                information.
+            dict: Dictionary that stores the created epoch in the key "epoch".
 
         Raises:
-            CommandFailure: if the daos pool query command fails.
+            CommandFailure: if the daos container create-snap command fails.
 
         """
-        return self._get_result(
+        self.result = self._get_result(
             ("container", "create-snap"), pool=pool, svc=svc, cont=cont,
-            sys_name=sys_name, snap=snap_name, epc=epoch, )
+            sys_name=sys_name, snap=snap_name, epc=epoch)
 
-    def container_destroy_snap(self, pool, cont, snap_name=None, epoch=None,
-                               svc=None, sys_name=None):
-        """Call daos container create-snap.
+        # Sample create-snap output.
+        # snapshot/epoch 1582610056530034697 has been created
+        data = {}
+        match = re.findall(
+            r"[A-Za-z\/]+\s([0-9]+)\s[a-z\s]+", self.result.stdout)
+        if match:
+            data["epoch"] = match[0]
+        return data
+
+    def container_destroy_snap(self, pool, cont, snap_name=None, epc=None,
+                               svc=None, sys_name=None, epcrange=None):
+        """Call daos container destroy-snap.
 
         Args:
             pool (str): Pool UUID.
             cont (str): Container UUID.
             snap_name (str, optional): Snapshot name.
-            epoch (str, optional): Epoch number.
+            epc (str): Epoch value of the snapshot to be destroyed.
             svc (str, optional): Pool service replicas, e.g., '1,2,3'. Defaults
                 to None.
             sys_name (str, optional): DAOS system name context for servers.
                 Defaults to None.
+            epcrange (list): If you want to use --epc, use a single-element
+                list. If you want to use --epcrange, use a list that contains
+                epoch for B in the first element and epoch for E in the second
+                element.
```

Comment:
I think this comment is now outdated.  It appears to reference the previous implementation.  I think it should look something like this now:
```suggestion
            epcrange (str): epoch range in the format "<start>-<end>".
```

Reply:
Dane